apps/users/urls/auth_urls.py

```diff
@@ -1,13 +1,31 @@
 from django.urls import path
 
 from apps.users.views.phone_verification_views import (
-    PhoneConfirmCodeView,
-    PhoneSendCodeView,
+    ChangePhoneConfirmCodeView,
+    ChangePhoneSendCodeView,
+    FindEmailConfirmCodeView,
+    FindEmailSendCodeView,
+    SignupConfirmCodeView,
+    SignupSendCodeView,
 )
 
 app_name = "phone_verifications"
 
 urlpatterns = [
-    path("phone/verifications/send-code", PhoneSendCodeView.as_view(), name="send_code"),
-    path("phone/verifications/confirm-code", PhoneConfirmCodeView.as_view(), name="confirm_code"),
+    path("phone/verifications/signup/send-code", SignupSendCodeView.as_view(), name="signup_send_code"),
+    path("phone/verifications/signup/confirm-code", SignupConfirmCodeView.as_view(), name="signup_confirm_code"),
+    path("phone/verifications/find-email/send-code", FindEmailSendCodeView.as_view(), name="find_email_send_code"),
+    path(
+        "phone/verifications/find-email/confirm-code",
+        FindEmailConfirmCodeView.as_view(),
+        name="find_email_confirm_code",
+    ),
+    path(
+        "phone/verifications/phone-change/send-code", ChangePhoneSendCodeView.as_view(), name="change_phone_send_code"
+    ),
+    path(
+        "phone/verifications/phone-change/confirm-code",
+        ChangePhoneConfirmCodeView.as_view(),
+        name="change_phone_confirm_code",
+    ),
 ]
```

Comment:
```suggestion
urlpatterns = [
    path("phone-verifications/signup/send-code", SignupSendCodeView.as_view(), name="signup_send_code"),
    path("phone-verifications/signup/confirm-code", SignupConfirmCodeView.as_view(), name="signup_confirm_code"),
    path("phone-verifications/find-email/send-code", FindEmailSendCodeView.as_view(), name="find_email_send_code"),
    path(
        "phone-verifications/find-email/confirm-code",
        FindEmailConfirmCodeView.as_view(),
        name="find_email_confirm_code",
    ),
    path(
        "phone-verifications/phone-change/send-code", ChangePhoneSendCodeView.as_view(), name="change_phone_send_code"
    ),
    path(
        "phone-verifications/phone-change/confirm-code",
        ChangePhoneConfirmCodeView.as_view(),
        name="change_phone_confirm_code",
    ),
]
```

이런식으로 구분하는게 좋을 것 같습니다

phone-verification이라는 휴대폰 인증 공통 리소스에 대해서 목적 행위별 엔드포인트 구분으로요